lightning/src/ln/channelmanager.rs

```diff
@@ -7579,6 +7619,121 @@ mod tests {
 		// Check that using the original payment hash succeeds.
 		assert!(inbound_payment::verify(payment_hash, &payment_data, nodes[0].node.highest_seen_timestamp.load(Ordering::Acquire) as u64, &nodes[0].node.inbound_payment_key, &nodes[0].logger).is_ok());
 	}
+
+	#[test]
+	fn test_id_to_peer_coverage() {
```

Comment:
Let me know if you'd like me to add the `by_id` map (`channel_by_id` in #1507), as well as the `short_to_chan_info` map to this test.

Reply:
Thanks! I'll look into that.